doc/api/crypto.md

```diff
@@ -378,7 +380,7 @@ The `decipher.setAuthTag()` method must be called before
 <!-- YAML
 added: v0.7.1
 -->
-- `autoPadding` {boolean} Defaults to `true`.
+- `autoPadding` {boolean} (Default `true`).
```

Comment:
If we're going to switch these, we should do so consistently throughout. There are other instances in this document.